docs/release-notes/Percona Everest 0.7.0 (2024-01-30).md

```diff
@@ -0,0 +1,29 @@
+# What's new in Percona Everest 0.7.0
+
+To begin your journey with Percona Everest, check out the [Quickstart Guide for Percona Everest](../quickstart-guide/quick-install.md).
+
+Percona Everest is an open source private database-as-a-service that helps developers deploy code faster, scale deployments rapidly, and reduce database administration overhead. Plus, you can regain control over your data, database configuration, and DBaaS costs.
+
+Version 0.7.0 introduces the following changes:
+
+## Beta release
+
+We're excited to announce that Percona Everest is now in Beta!
+
+Under development for the past five months and in testing since October 2023, we’re now taking Percona Everest public, making it open for anyone who wants to explore our Cloud Native database platform.
+
```

Comment:
Missing a mention to the fact that that we added a UI page for managing the monitoring endpoints.